<Type Name="XamlObjectReaderException" FullName="System.Xaml.XamlObjectReaderException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="ea5dc71151e3bcadfcf6c1e7f9c32b489f4d8e22" />
    <Meta Name="ms.sourcegitcommit" Value="d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="de-DE" />
    <Meta Name="ms.lasthandoff" Value="04/03/2018" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class XamlObjectReaderException : System.Xaml.XamlException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit XamlObjectReaderException extends System.Xaml.XamlException" />
  <TypeSignature Language="DocId" Value="T:System.Xaml.XamlObjectReaderException" />
  <TypeSignature Language="VB.NET" Value="Public Class XamlObjectReaderException&#xA;Inherits XamlException" />
  <TypeSignature Language="C++ CLI" Value="public ref class XamlObjectReaderException : System::Xaml::XamlException" />
  <AssemblyInfo>
    <AssemblyName>System.Xaml</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Xaml.XamlException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
      <span data-ttu-id="cd01f-101">Die Ausnahme, die von mehreren internen <see cref="T:System.Xaml.XamlObjectReader" />-Hilfe-APIs ausgelöst wird.</span>
      <span class="sxs-lookup">
        <span data-stu-id="cd01f-101">The exception that is thrown by several <see cref="T:System.Xaml.XamlObjectReader" /> internal helper APIs.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="cd01f-102">Initialisiert eine neue Instanz der <see cref="T:System.Xaml.XamlObjectReaderException" />-Klasse.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-102">Initializes a new instance of the <see cref="T:System.Xaml.XamlObjectReaderException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlObjectReaderException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlObjectReaderException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlObjectReaderException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="cd01f-103">Initialisiert eine neue Instanz der <see cref="T:System.Xaml.XamlObjectReaderException" />-Klasse mit einer vom System generierten Meldung, die den Fehler beschreibt.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-103">Initializes a new instance of the <see cref="T:System.Xaml.XamlObjectReaderException" /> class with a system-supplied message that describes the error.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="cd01f-104">Dieser Konstruktor initialisiert die <xref:System.Exception.Message%2A>-Eigenschaft der neuen Instanz mit einer vom System gelieferten Meldung, die den Fehler beschreibt.</span><span class="sxs-lookup"><span data-stu-id="cd01f-104">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlObjectReaderException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlObjectReaderException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlObjectReaderException(System::String ^ message);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="cd01f-105">Die Meldung, in der die Ausnahme beschrieben wird</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-105">The message that describes the exception.</span>
          </span>
          <span data-ttu-id="cd01f-106">Der Aufrufer dieses Konstruktors muss sicherstellen, dass diese Zeichenfolge für die aktuelle Systemkultur lokalisiert wurde.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-106">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="cd01f-107">Initialisiert eine neue Instanz der <see cref="T:System.Xaml.XamlObjectReaderException" />-Klasse mit einer angegebenen Meldung, die den Fehler beschreibt.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-107">Initializes a new instance of the <see cref="T:System.Xaml.XamlObjectReaderException" /> class with a specified message that describes the error.</span>
          </span>
        </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected XamlObjectReaderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlObjectReaderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; XamlObjectReaderException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <span data-ttu-id="cd01f-108">Das Objekt, das die Daten des serialisierten Objekts enthält.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-108">The object that holds the serialized object data.</span>
          </span>
        </param>
        <param name="context">
          <span data-ttu-id="cd01f-109">Die Kontextinformationen zur Quelle bzw. zum Ziel.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-109">The contextual information about the source or destination.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="cd01f-110">Initialisiert eine neue Instanz der <see cref="T:System.Xaml.XamlObjectReaderException" />-Klasse mit serialisierten Daten.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-110">Initializes a new instance of the <see cref="T:System.Xaml.XamlObjectReaderException" /> class with serialized data.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="cd01f-111">Dieser Konstruktor wird während der Deserialisierung, das über einen Stream übertragene Ausnahmeobjekt aufgerufen.</span><span class="sxs-lookup"><span data-stu-id="cd01f-111">This constructor is called during deserialization to reconstitute the exception object that is transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public XamlObjectReaderException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Xaml.XamlObjectReaderException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; XamlObjectReaderException(System::String ^ message, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Xaml</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="cd01f-112">Die Meldung, in der die Ausnahme beschrieben wird</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-112">The message that describes the exception.</span>
          </span>
          <span data-ttu-id="cd01f-113">Der Aufrufer dieses Konstruktors muss sicherstellen, dass diese Zeichenfolge für die aktuelle Systemkultur lokalisiert wurde.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-113">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="cd01f-114">Die Ausnahme, die die Ursache der aktuellen Ausnahme ist.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-114">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="cd01f-115">Wenn der <c>innerException</c>-Parameter nicht <see langword="null" /> ist, wird die aktuelle Ausnahme in einem <see langword="catch" />-Block ausgelöst, der die innere Ausnahme behandelt.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-115">If the <c>innerException</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="cd01f-116">Initialisiert eine neue Instanz der <see cref="T:System.Xaml.XamlObjectReaderException" />-Klasse mit einer angegebenen Fehlermeldung und einem Verweis auf die innere Ausnahme, die diese Ausnahme ausgelöst hat.</span>
          <span class="sxs-lookup">
            <span data-stu-id="cd01f-116">Initializes a new instance of the <see cref="T:System.Xaml.XamlObjectReaderException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="cd01f-117">Eine Ausnahme, die als direktes Ergebnis einer vorhergehenden Ausnahme ausgelöst wird, muss in der <xref:System.Exception.InnerException%2A>-Eigenschaft über einen Verweis auf die vorhergehende Ausnahme verfügen.</span><span class="sxs-lookup"><span data-stu-id="cd01f-117">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="cd01f-118">Die <xref:System.Exception.InnerException%2A> -Eigenschaft gibt denselben Wert, der an den Konstruktor; übergeben wird, oder gibt `null` Wenn die <xref:System.Exception.InnerException%2A> -Eigenschaft nicht den Wert der inneren Ausnahme an den Konstruktor bereitstellt.</span><span class="sxs-lookup"><span data-stu-id="cd01f-118">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor; or it returns `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>